test/Conversion/TorchToLinalg/gridsampler.mlir

```diff
@@ -0,0 +1,68 @@
+// RUN: torch-mlir-opt <%s -convert-torch-to-linalg -split-input-file -verify-diagnostics | FileCheck %s
+
+// CHECK: #map
+// CHECK-LABEL: func @grid_sampler
+// CHECK:  %0 = torch_c.to_builtin_tensor %arg0 : !torch.vtensor<[4,10,10,4],f32> -> tensor<4x10x10x4xf32>
```

Comment:
You will want to check how to do regex matching. Shortened example below but it lets us do `CHECK-DAG` instead of `CHECK`, removing the ordering restrictions:
```
// CHECK-LABEL: func @grid_sampler
// CHECK:  %[[ARG0:.+]] = torch_c.to_builtin_tensor %arg0 : !torch.vtensor<[4,10,10,4],f32> -> tensor<4x10x10x4xf32>
// CHECK:  %[[C2:.+]] = arith.constant 2 : index
// CHECK:  %[[DIM2:.+]] = tensor.dim %0, %c2_3 : tensor<4x10x10x4xf32>
// CHECK:  %[[C3:.+]] = arith.constant 3 : index
// CHECK:  %[[DIM3:.+]] = tensor.dim %0, %c3 : tensor<4x10x10x4xf32>
```

You can reuse names sometimes if values are only used in limited scope but it avoids MLIR core changes from breaking your tests. E.g. code reordering is technically allowed between versions.